.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: Lint
+
+on: [push]
```

I'm not sure that running this on every push is useful. I would suggest running linters against PRs and master branch pushes. 

```suggestion
on:
  pull_request:
  push:
    branches: [master]
```